Sources/AWSLambdaRuntimeCore/Lambda.swift

```diff
@@ -53,30 +32,23 @@ public enum Lambda {
         return String(cString: value)
     }
 
-    #if compiler(>=5.5) && canImport(_Concurrency)
-    // for testing and internal use
-    @available(macOS 12, iOS 15, tvOS 15, watchOS 8, *)
-    internal static func run<Handler: LambdaHandler>(configuration: Configuration = .init(), handlerType: Handler.Type) -> Result<Int, Error> {
-        self.run(configuration: configuration, factory: { context -> EventLoopFuture<ByteBufferLambdaHandler> in
-            let promise = context.eventLoop.makePromise(of: ByteBufferLambdaHandler.self)
-            promise.completeWithTask {
-                try await Handler(context: context)
-            }
-            return promise.futureResult
-        })
-    }
-    #endif
-
-    // for testing and internal use
-    internal static func run(configuration: Configuration = .init(), factory: @escaping HandlerFactory) -> Result<Int, Error> {
-        let _run = { (configuration: Configuration, factory: @escaping HandlerFactory) -> Result<Int, Error> in
+    /// Run a Lambda defined by implementing the ``ByteBufferLambdaHandler`` protocol.
+    /// The Runtime will manage the Lambdas application lifecycle automatically. It will invoke the
+    /// ``ByteBufferLambdaHandler/factory(context:)`` to create a new Handler.
+    ///
+    /// - parameters:
+    ///     - factory: A `ByteBufferLambdaHandler` factory.
+    ///
+    /// - note: This is a blocking operation that will run forever, as its lifecycle is managed by the AWS Lambda Runtime Engine.
+    internal static func run<Handler: ByteBufferLambdaHandler>(configuration: Configuration = .init(), handlerType: Handler.Type) -> Result<Int, Error> {
```

Comment:
is `handlerType` used?

Reply:
I guess needed for the generics?

Reply:
Yes. It is there to make the user declare the Generic type. Otherwise the user would need to write:
```swift
Lambda.run<Handler>()
```